cartography/models/aws/inspector/packages.py

```diff
@@ -0,0 +1,73 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import OtherRelationships
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class AWSInspectorPackageNodeProperties(CartographyNodeProperties):
+    id: PropertyRef = PropertyRef('id')
+    region: PropertyRef = PropertyRef('Region', set_in_kwargs=True)
+    awsaccount: PropertyRef = PropertyRef('AWS_ID', set_in_kwargs=True)
+    findingarn: PropertyRef = PropertyRef('findingarn')
+    name: PropertyRef = PropertyRef('name')
+    arch: PropertyRef = PropertyRef('arch')
+    version: PropertyRef = PropertyRef('version')
+    release: PropertyRef = PropertyRef('release')
+    epoch: PropertyRef = PropertyRef('epoch')
+    manager: PropertyRef = PropertyRef('packageManager')
+    filepath: PropertyRef = PropertyRef('filePath')
+    fixedinversion: PropertyRef = PropertyRef('fixedInVersion')
+    sourcelayerhash: PropertyRef = PropertyRef('sourceLayerHash')
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
```

Comment:
Add indexes for 

1. findingarn
2. name
4. version
5. release

Reply:
Added for findingarn and name

Reply:
I think they'll be useful for manual queries when we want to investigate.

Reply:
Added